backend/src/models/open_content.go

```diff
@@ -42,6 +42,28 @@ type OpenContentUrl struct {
 
 func (OpenContentUrl) TableName() string { return "open_content_urls" }
 
+type CombinedFavorite struct {
```

Comment:
Can we call this `OpenContentFavorites`? 

We have Programs being favorited as well so that is a bit more explicit